### DISCUSSION TITLE
Add medals for having played with certain mod combinations

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#13868] [Ideas]
You can make many variations of interaction with new mods, for which you can give medals. Here you can describe them, I think.

UPD for march 2022. There are a couple more suggestions (so as not to make a separate discussion).

At the moment, the medal does not have any clues on what to do to get it. I think this needs to be fixed and the properties of the medal should be added to what exactly the player did to obtain it.

In this case, we have a more transparent motivation to become better, as well as an indicator for others - what exactly you have achieved.

Also, as mentioned above, you can add the "rarity" property of the medal. Show how many percent of the players received this medal.

**Comment by frenzibyte** [comment]
Sounds like they should be part of the medals system instead? unlocking medals when playing with "unusual" mod combinations or otherwise.

**Reply by Moltanicaa** [reply]
Achievements = medals? 

We need new medals for new mods

**Reply by smoogipoo** [reply]
> Achievements = medals?

Yes, I've adjusted the title.
